apps/web/pages/api/example.ts

```diff
@@ -0,0 +1,267 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { cardExamples } from "@pages/api/plain/example-cards";
+import crypto from "crypto";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { prisma } from "@calcom/prisma";
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  try {
+    // HMAC verification
+    const requestBody = JSON.stringify(req.body);
+    const incomingSignature = req.headers["plain-request-signature"];
+    const expectedSignature = crypto
+      .createHmac("sha-256", process.env.HMAC_SECRET_KEY!)
+      .update(requestBody)
+      .digest("hex");
+
+    if (incomingSignature !== expectedSignature) {
+      return res.status(403).send("Forbidden");
+    }
+
+    // Disabled vercel's edge cache
+    res.setHeader("Cache-Control", "no-cache");
+
+    const { cardKeys, customer } = req.body;
+    const { email, id, username, timeZone, emailVerified, plan, identityProvider, twoFactorEnabled } =
+      customer;
+
+    if (!email) {
+      return res.status(400).json({ error: "Invalid customer data" });
+    }
+
+    // Fetch user details from the database using Prisma
+    const user = await prisma.user.findUnique({
+      where: { email },
```

Comment:
Let's only fetch certain fields based on what we need. The user table has a lot of columns nowadays